velox/expression/SimpleFunctionAdapter.h

```diff
@@ -235,7 +244,16 @@ class SimpleFunctionAdapter : public VectorFunction {
           applyContext, nextNonNull, packed, readers..., oneReader);
     } else {
       auto* oneUnpacked = packed.at(POSITION);
-      auto oneReader = VectorReader<arg_at<POSITION>>(oneUnpacked);
+      auto oneReader = VectorReader<arg_at<POSITION>>(
+          oneUnpacked, FUNC::udf_has_callNullFree);
+
+      if constexpr (FUNC::udf_has_callNullFree) {
```

Comment:
is this comment needed here to line 291

Reply:
what do we do if !is_default_contains_nulls_behavior
we mentioned what we do not do. we call call right or callNullable?

Reply:
That's right.